influx/templates_test.go

```diff
@@ -283,7 +283,7 @@ func TestGroupByVarString(t *testing.T) {
 				ReportingInterval: 10 * time.Second,
 				Duration:          time.Hour,
 			},
-			want: "time(1s)",
+			want: "time(107ms)",
```

Comment:
test is supposed to check for the minimum of 1 second
... also we'll need a test for less than one millisecond.